src/formatters/junit.js

```diff
@@ -0,0 +1,42 @@
+/*eslint no-console: "off"*/
+
+function printResults(results) {
+  const testCases = results.map(result => {
+    const file = result.filePath;
+    return {
+      _attributes: {
+        name : file
+      },
+      error : result.errors.map(error => {
+        return {
+          _attributes: {
+            message: `${error.message}`,
+            type: 'gherkin-lint-error'
+          },
+          _cdata: `${file}:${error.line} (${error.rule}) ${error.message}`
+        };
+      })
+    };
+  });
+  const testSuiteReport = {
+    _declaration:{
+      _attributes: {
+        version: '1.0',
+        encoding: 'utf-8'
+      }
+    },
+    testsuite : {
+      _attributes: {
+        name: 'gherkin-lint',
+      },
+      testcase : testCases
+    }
+  };
+  let convert = require('xml-js');
+  const junitXml = convert.js2xml(testSuiteReport, {compact: true, spaces: 4});
```

Comment:
might be wise to wrap in the try/catch just in case is blows up

Reply:
what should I catch? Is it not better to just let it bubble up since it's something I can't recover from anyway?

Reply:
It's up to you, if you're ok with that then yes.